src/wagtail_tinytableblock/utils.py

```diff
@@ -11,22 +12,28 @@
 Cell = Literal["td", "th"]
 
 
-def sanitise_html(content: str) -> str:
+def sanitise_html(content: str, *, allow_links: bool = False) -> str:
+    tags: set[str] = {"table", "tr", "th", "td", "thead", "tbody", "caption"}
+    attributes: dict[str, set[str]] = {
+        "*": {"class"},
+        "th": {"colspan", "rowspan", "align", "scope"},
+        "td": {"colspan", "rowspan", "align", "scope"},
+    }
+    if allow_links:
+        tags |= {"a"}
+        attributes["a"] = {"href", "rel", "title"}
+
     return nh3.clean(  # pylint: disable=no-member
-        content,
-        tags={"table", "tr", "th", "td", "thead", "tbody", "a", "caption"},
-        attributes={
-            "*": {"class"},
-            "th": {"colspan", "rowspan", "align", "scope"},
-            "td": {"colspan", "rowspan", "align", "scope"},
-            "a": {"href", "rel"},
-        },
+        unescape(content),
```

Comment:
Question: Where does the escaping come from?

Reply:
If you paste non-allowed tags, they get escaped by TinyMCE so they may not get sanitised

Reply:
Oh, interesting, nice catch!